client/modules/accounts/templates/profile/profile.js

```diff
@@ -1,6 +1,7 @@
-import * as Collections from "/lib/collections";
 import { Meteor } from "meteor/meteor";
 import { Template } from "meteor/templating";
+import * as Collections from "/lib/collections";
+import { Marketplace } from "/imports/plugins/included/marketplace/lib/api";
```

Comment:
Agreed it's better to keep them separate, this is why a `Reaction.Marketplace` solution for methods is also problematic in some ways. Some functions tied around the marketplace need to live in Reaction, at least the core ones that check sellerId, sellerShop and the core functionality that checks if the module is enabled and the user is a guest, very simple functions. They're not heavy and probably they can be consolidated with existing functions, moving forward.
Should we place it along with the other two functions in core.jshttps://github.com/reactioncommerce/reaction/pull/1758/files/9f64febe5e71033b5691ed92946ed52c7c3db53f#diff-541cd596a355327d6884076c3ee12121?
Although the current /import solution is clean, it is not practical. Whereas putting marketplace core functions in core.js to check shopIds and permissions when the module is enabled is probably less clean but more practical. This way the module can be deleted and disabled.